docs/etcd/demo_cluster_setup.md

```diff
@@ -0,0 +1,272 @@
+# Set up a demo etcd cluster
```

Comment:
Seems good, thank you :) I will have to test it as well because I need to see how the authentication is done there to be able to sync that with the operator.

I wanted to give a way to use the existing etcd on kubernetes or another one, but it involves certificates which I didn't implement because I wanted it to be available quickly

Reply:
See PR #42